Sources/EmbraceConfigInternal/EmbraceConfig.swift

```diff
@@ -30,6 +32,7 @@ public class EmbraceConfig {
         self.notificationCenter = notificationCenter
         self.logger = logger
         self.configurable = configurable
+        self.queue = DispatchQueue(label: "com.embrace.config", attributes: .concurrent)
 
         update()
```

Comment:
We should move this call to update as it is a side effect of the `init` method.

Think it makes more sense if we call this in the `Embrace.start` method in the [async job here](https://github.com/embrace-io/embrace-apple-sdk/pull/101/files#diff-b90b0a3a8c473dbc673660227e7cab7328e5201df828de2a1e60c7b2431c4edaR210)